src/components/community/news-item/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC, useState } from 'react';
+import { theme, Colour } from '@fellesdatakatalog/theme';
+import SC from './styled';
+import TimeStamp from '../time-stamp';
+import TruncatedText from '../../truncated-text';
+import { PATHNAME_NEWS_ARTICLE } from '../../../constants/constants';
+
+interface Props {
+  created: string;
+  field_ingress: string;
+  title: string;
+  id?: string;
+  customColor?: string;
+  customHoverColor?: string;
+}
+
+const PostLink: FC<Props> = ({
+  created,
+  field_ingress,
+  title,
+  id,
+  customColor = (() =>
+    theme.colour(Colour.NEUTRAL, 'N10')) as unknown as string,
```

Comment:
hvorfor casting-mellomsteget til `unknown`?

Reply:
Godt observasjon! Dette er fordi TypeScript klager hvis man caster direkte til teskts streng, fordi typen som skal castes fra overlapper ikke med typen `string`.
`"Conversion of type '() => any' to type 'string' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first"`

Reply:
:raised_eyebrow: